docs/source/tutorials/quickstart.rst

```diff
@@ -154,13 +159,13 @@ Our scene is a little basic, so let's add some bells and whistles.  Modify the
            self.play(Transform(square, circle))  # interpolate the square into the circle
            self.play(FadeOut(square))  # fade out animation
 
-And render it using the following command:
+2. Render ``SquareToCircle`` using the following command:
 
 .. code-block:: bash
 
    manim -pql scene.py SquareToCircle
 
-The output should look as follows.
+The following animation displays:
```

Comment:
```suggestion
The following animation should be rendered:
```

Reply:
Would

> The following animation will render:

be sufficient? I feel that "should be" is less confident than "will" and a guide shouldn't be written in maybes.